libs/application/templates/aosh/work-accident-notification/src/fields/PdfOverview/pdfDocument.tsx

```diff
@@ -0,0 +1,126 @@
+    application.externalData,
+    formatMessage,
+  )
+  const accidentInfo = getAccidentInformationForOverview(
+    application.answers,
+    application.externalData,
+    formatMessage,
+  )
+  return (
+    <Document>
+      <Page size="A4" style={pdfStyles.body}>
+        <View style={pdfStyles.listInfo}>
+          <Text style={pdfStyles.title}>
+            {formatMessage(accident.about.pageTitle)}
+          </Text>
+          <View style={pdfStyles.view}>
+            <Text style={pdfStyles.header}>
+              {formatMessage(information.labels.company.pageTitle)}
+            </Text>
+            {companyInfo.map((info) => {
+              return <Text style={pdfStyles.text}>{info}</Text>
+            })}
+            <Text style={pdfStyles.header}>
+              {formatMessage(sections.draft.accident)}
+            </Text>
+            {accidentInfo.map((info) => {
+              return <Text style={pdfStyles.text}>{info}</Text>
+            })}
+          </View>
+          <Text style={pdfStyles.title}>
+            {formatMessage(overview.employee.employees)}
+          </Text>
+          {employees.map((employee, index) => {
+            const employeeInfo = getEmployeeInformationForOverview(
+              application.externalData,
+              employee,
+              formatMessage,
+            )
+            const causeandConsequencesInfo = getCauseAndConsequencesForOverview(
+              application.externalData,
+              application.answers,
+              index,
+              formatMessage,
+            )
+            return (
+              <View key={`employee-${index}`} style={pdfStyles.view}>
+                <Text style={pdfStyles.employeeHeader}>
+                  {employee.nationalField.name}
+                </Text>
+                <Text style={pdfStyles.header}>
+                  {formatMessage(overview.labels.employee)}
+                </Text>
+                {employeeInfo.map((info) => {
+                  return <Text style={pdfStyles.text}>{info}</Text>
+                })}
+                <Text style={pdfStyles.header}>
+                  {formatMessage(overview.labels.events)}
+                </Text>
+                {causeandConsequencesInfo.map((info) => {
+                  return <Text style={pdfStyles.text}>{info}</Text>
+                })}
+              </View>
+            )
+          })}
+        </View>
+      </Page>
+    </Document>
+  )
+}
+
+export const pdfStyles = StyleSheet.create({
+  body: {
+    paddingTop: 40,
+    paddingBottom: 70,
+    paddingHorizontal: 60,
+    fontSize: 10,
+  },
+  title: {
+    fontSize: 24,
+    marginBottom: 16,
+  },
+  employeeHeader: {
+    fontSize: 18,
+    fontWeight: 600,
+    marginBottom: 5,
+  },
+  header: {
+    fontSize: 14,
+    fontWeight: 600,
+    marginTop: 12,
+    marginBottom: 5,
+  },
+  text: {
+    marginBottom: 4,
+  },
+  listInfo: {
+    paddingBottom: 30,
+  },
+  view: {
+    marginBottom: 20,
+  },
+})
```

Comment:
Should this be in the `PdfOverview.css.ts` file? This is exported here but I don't see it imported anywhere on a quick glance

Reply:
It's using stylesheet from react-pdf library, but it should not be exported.

Reply:
It is not a vanilla-extract css and therefore I get an error if I move it to css.ts, but it should not be exported, so I'll remove that.